CMakeLists.txt

```diff
@@ -144,10 +144,6 @@ if (HAVE_TCP_KEEPIDLE AND HAVE_TCP_KEEPINTVL AND HAVE_TCP_KEEPCNT)
   set(SYSLOG_NG_HAVE_TCP_KEEPALIVE_TIMERS 1)
 endif()
 
-if (NOT APPLE)
-  set(SYSLOG_NG_HAVE_ENVIRON 1)
-endif()
-
```

Comment:
please remove the according block from the automake configure file `configure.ac` as well

Reply:
Done